model/aro_hcp/v1alpha1/autoscaler_resource.model

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages a specific cluster autoscaler.
```

Comment:
The `clusters_mgmt` version also supports POST and DELETE. I believe this is required here as well. @miguelsorianod PCMIIW